GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -6,4 +6,21 @@
 //  Copyright © 2023 Shalva Avanashvili. All rights reserved.
 //
 
+import SwiftUI
 import Foundation
+
+class DogBreedsListViewModel: ObservableObject {
+    var body: some View {
```

Comment:
Viewmodel does not have anything to do with views. So we need to delete the `body`. Lets leave this class empty for now.